source/opt/ir_context.h

```diff
@@ -625,13 +636,23 @@ class IRContext {
   // the function that contains |bb|.
   bool IsReachable(const opt::BasicBlock& bb);
 
+  // Return the stage of the module. Will generate error if entry points don't
+  // all have the same stage.
+  uint32_t GetStage() const;
```

Comment:
Yes, but stage is more specific and in my mind more apt because a stage is a piece of a pipeline and this is ultimately a pipeline optimization. So I would prefer to keep this.